test/common/grafana_routes.go

```diff
@@ -78,8 +73,7 @@ func TestGrafanaExternalRouteDashboardExist(t *testing.T, ctx *TestingContext) {
 	if dashboardResp.StatusCode != http.StatusOK {
 		dumpResp, _ := httputil.DumpResponse(dashboardResp, true)
 		t.Logf("dumpResp: %q", dumpResp)
-		t.Skipf("skipping due to known flaky behaviour https://issues.redhat.com/browse/INTLY-6738, got status : %v", dashboardResp.StatusCode)
-		// t.Fatalf("unexpected status code on success request, got=%+v", dashboardResp)
+		t.Fatalf("unexpected status code on success request, got=%+v", dashboardResp)
```

Comment:
Makes sense to remove the skip given the flakey login part has been replaced with using a token.